parser/tests/fixtures/tokenizer/validator_registration.v.py.json

```diff
@@ -530,85 +530,85 @@ def deposit(pubkey: bytes[PUBKEY_LENGTH],
       4,
       24
     ],
-    "line": "PUBKEY_LENGTH: constant(uint256) = 48  # bytes\n"
+    "line": "PUBKEY_LENGTH: constant(u256) = 48  # bytes\n"
   },
   {
     "typ": "NAME",
-    "string": "uint256",
+    "string": "u256",
     "start": [
       4,
       24
     ],
     "end": [
       4,
-      31
+      28
     ],
-    "line": "PUBKEY_LENGTH: constant(uint256) = 48  # bytes\n"
+    "line": "PUBKEY_LENGTH: constant(u256) = 48  # bytes\n"
   },
   {
     "typ": "OP",
     "string": ")",
     "start": [
       4,
-      31
+      28
     ],
     "end": [
       4,
-      32
+      29
     ],
-    "line": "PUBKEY_LENGTH: constant(uint256) = 48  # bytes\n"
+    "line": "PUBKEY_LENGTH: constant(u256) = 48  # bytes\n"
   },
   {
     "typ": "OP",
     "string": "=",
     "start": [
       4,
-      33
+      30
     ],
     "end": [
       4,
-      34
+      31
     ],
-    "line": "PUBKEY_LENGTH: constant(uint256) = 48  # bytes\n"
+    "line": "PUBKEY_LENGTH: constant(u256) = 48  # bytes\n"
   },
   {
     "typ": "NUMBER",
     "string": "48",
     "start": [
       4,
-      35
+      32
     ],
     "end": [
       4,
-      37
+      34
     ],
-    "line": "PUBKEY_LENGTH: constant(uint256) = 48  # bytes\n"
+    "line": "PUBKEY_LENGTH: constant(u256) = 48  # bytes\n"
   },
   {
     "typ": "COMMENT",
     "string": "# bytes",
     "start": [
       4,
-      39
+      36
```

Comment:
For the record: I updated this file by hand but by hand. 33 % through I thought to myself: "There's probably an easy way to just regenerate it", 66 % I was almost ready to give up and look into regenerating but then followed through. :sweat_smile:

Reply:
:pray: